src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
```

Comment:
Modifying `Array.prototype` is generally not recommended as it can lead to conflicts and unexpected behavior in other parts of the code. Consider creating a utility function instead.